test/nuctl_build_test.go

```diff
@@ -18,18 +18,21 @@ package main
 
```

Comment:
This module is obsolete. All build tests should be implemented in `processor/build/runtime/<runtime>/test` - that's where all the other tests are. nuctl tests will be written in nuctl when we have an e2e platform (including build). If you feel this is not a runtime-specific test you can make a `processor/build/test` module... but I prefer to test as close to the source as possible even at the expense of some overlapping tests

Reply:
OK